### DISCUSSION TITLE
IBAN for Donations

### DISCUSSION THREAD

**Discussion by ValValetl** [Zettlr/Zettlr#4130] [General]
hej there, I am just testing your app for the Zettelkasten principle. Looks really nice, is there a way to support you via a bank account donation? I saw the Patreon page, but prefer to donate directly, as usually, more of the money gets to the receiver.

**Comment by nathanlesage** [comment]
You can make a one-time donation via PayPal if you'd like: https://www.paypal.me/hendrikerz

**Reply by ValValetl** [reply]
thanks for the reply, in this case, I will join on patreon :)
